### PR TITLE
Removed obsolete configurations

### LINKED ISSUE #2405: Removing obsolete configurations
<b>Actual Behaviour</b>

Few configurations such as "compile" , "androidTestCompile","debugCompile" ,"releaseCompile" , "testCompile" have become obsolete now, but are still present in build.gradle.

<b>Expected Behaviour</b>

These configurations should be replaced with their new names , such as "compile" should be replaced with "implementation" , 'androidTestCompile' should be replaced with 'androidTestImplementation'.

<b>Would you like to work on the issue?</b>

Yes

### PR DESCRIPTION
**Pull request by Priyanka488**
Fixed #2405 

Changes: app/build.gradle , build.gradle , openCVLibrary24133/src/main/AndroidManifest.xml .
Changed 'compile' to 'implementation'
               'androidTestCompile' to androidTestImplementation'
               'debugCompile' to 'debugImplementation'
            'releaseCompile' to 'releaseImplementation'

### CODE REVIEW

File: openCVLibrary24133/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -4,5 +4,5 @@
       android:versionCode="24133"
       android:versionName="2.4.13.3">
 
-    <uses-sdk android:minSdkVersion="8" />
```
Comment (abishekvashok) [inline openCVLibrary24133/src/main/AndroidManifest.xml]: Why was this removed?

Comment (Priyanka488) [reply openCVLibrary24133/src/main/AndroidManifest.xml]: Actually there were some build errors so had to remove this. I'll try doing it again.
